### DISCUSSION TITLE
ch3-asm/ch3-06-func-again

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#619] [General]
# ch3-asm/ch3-06-func-again

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch3-asm/ch3-06-func-again.html

**Comment by dayun1999** [comment]
汇编这块很有收获，感谢！！！

**Comment by alpha-baby** [comment]
go1.17 函数调用规范已经更新了，可以参考:https://xargin.com/go1-17-new-calling-convention/

**Comment by shapled** [comment]
3.6.4 方法函数  中的例子无法运行，提示
makeABIWrapper support for wrapping methods not implemented

go 版本 1.23.6

**Comment by changkaiyan** [comment]
您好，我将尽快回复您的邮件。如果您的邮件无需回复，则收到本回复表明我已收到您的邮件。请勿回复，谢谢合作。This is an automatic reply, confirming that your e-mail was received.Thank you.
